src/errors.ts

```diff
@@ -11,3 +11,4 @@ Error: Dart not found
 
 Please ensure that Dart is installed and its binary is added to your system's PATH environment variable. You can download and install Dart from the official website: https://dart.dev/get-dart. Once installed, try running the command again.
 `;
+export const GENEZIO_DARTAOTRUNTIME_NOT_FOUND = `We couldn't find the "dartaotruntime" executable. Make sure it is probably installed. Check the Dart documentation for more information: https://dart.dev/get-dart.`;  
```

Comment:
```suggestion
export const GENEZIO_DARTAOTRUNTIME_NOT_FOUND = `
Error: `dartaotruntime` not found

Please ensure that `dartaotruntime` is installed and its binary is added to your system's PATH environment variable. Check the Dart documentation for more information: https://dart.dev/get-dart.`;

```

Reply:
In my case, `dartaotruntime` was installed, but it wasn't in PATH. A hint to check that could be useful for the user.